src/toil/fileStores/__init__.py

```diff
@@ -14,6 +14,8 @@
 import os
 import stat
 
+from uuid import uuid4
+
 
 def make_public_dir(dirName: str) -> str:
```

Comment:
I don't think that, after this PR, `make_public_dir` is ever called anymore. We should remove it if it is no longer used.